doc/chapters/debugging/index.rst

```diff
@@ -120,11 +120,23 @@ able to connect to the process inside the container.
 
 Core dumps
 ----------
+`SoftwareContainer` is configured to make the commands or functions that run inside the container
+create a core dump in case of, for example, a segmentation fault.
+
+The naming of a core dump file is the same inside a container as on the host, meaning this can
+be set by looking at `/proc/sys/kernel/core_pattern` or by running `sysctl kernel.core_pattern`.
+Because the container and host share a kernel, the setting on the host will be visible in the
+container.
+
+Note that the core_pattern can contain a path to where to store core dumps, so a wise decision from
+a systems perspective, would be to set it to some directory which is then mounted into each
+container, either by issuing `BindMount` calls on D-Bus, or by adding it as a FileGateway
```

Comment:
Imho, the pattern should put the core dump in the CWD and execute from a directory with RW access. Setting a specific path will wreak havoc on apps that might crash outside containers.